notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(f"[PT]: {pt_transcription[0]}")
</pre><p>It's not easy to understand what's that at a glance. I'd recommend to use full names: Ground Truth, PyTorch, OpenVINO. Or, at least, add a description above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>updated</p>